pkg/catalog/manager/manager.go

```diff
@@ -29,12 +29,17 @@ type Manager struct {
 	httpClient            http.Client
 	uuid                  string
 	catalogClient         v3.CatalogInterface
```

Comment:
Nit: lets rearrange the order, so all the catalog related clients/listers are next to each other